living/serbian.md

```diff
@@ -5,41 +5,73 @@ Additional names:
 - 
 
 News:
-- 
-- 
+- https://www.b92.net/ (Daily)
+- https://www.telegraf.rs/
+- https://www.n1info.rs/
+- https://www.blic.rs/ (Daily)
+- https://www.politika.rs/ (Daily)
+- https://www.danas.rs/ (Daily)
+- https://www.novosti.rs/
+- https://www.kurir.rs/ (Tabloid)
+- https://www.alo.rs/ (Tabloid)
+- https://www.nova.rs/
+- https://www.rts.rs/ (Serbian public broadcaster)
 
 Culture / History:
-- 
-- 
+- https://www.narodnimuzej.rs/ (National Museum of Serbia)
+- https://www.etnografskimuzej.rs/ (Ethnographic Museum in Belgrade)
+- https://www.sanu.ac.rs/ (Serbian Academy of Sciences and Arts)
+- https://www.matica.rs/ (Matica Srpska - Serbian cultural and scientific institution)
+- https://muzej-jugoslavije.org/ (Museum of Yugoslavia)
+- https://www.heritage.gov.rs/latinica/index.php (Republic Institute for the Protection of Cultural Monuments)
 
 Government:
-- 
-- 
+- https://www.srbija.gov.rs/ (Official website of the Government of the Republic of Serbia)
+- https://www.predsednik.rs/ (Official website of the President of the Republic of Serbia)
+- http://www.parlament.gov.rs/ (National Assembly of the Republic of Serbia)
+- https://www.mfa.gov.rs/ (Ministry of Foreign Affairs)
+- https://www.mup.gov.rs/ (Ministry of Internal Affairs)
+- https://www.mfin.gov.rs/ (Ministry of Finance)
+- https://www.privreda.gov.rs/ (Ministry of Economy)
+- https://www.ekologija.gov.rs/ (Ministry of Environmental Protection)
+- https://www.mei.gov.rs/ (Ministry of European Integration)
+- https://www.mod.gov.rs/ (Ministry of Defence)
+- https://www.ite.gov.rs/ (Office for Information Technologies and eGovernment - includes the eUprava Portal for e-services)
 
 Political Parties:
-- 
-- 
+- https://sr.wikipedia.org/wiki/%D0%A1%D0%BF%D0%B8%D1%81%D0%B0%D0%BA_%D0%BF%D0%BE%D0%BB%D0%B8%D1%82%D0%B8%D1%87%D0%BA%D0%B8%D1%85_%D1%81%D1%82%D1%80%D0%B0%D0%BD%D0%B0%D0%BA%D0%B0_%D1%83_%D0%A1%D1%80%D0%B1%D0%B8%D1%98%D0%B8 (list of serbain Political Parties)
```

Comment:
Typo here (list of _Serbian_ [...]). Also, there are a great many political parties listed on this page, and only some of them are added here in the lines below.  Could you add all of them?